cpp/src/common/cumlHandle.cpp

```diff
@@ -143,6 +143,10 @@ void cumlHandle::setStream(cudaStream_t stream) { _impl->setStream(stream); }
 
 cudaStream_t cumlHandle::getStream() const { return _impl->getStream(); }
 
+const cudaDeviceProp& cumlHandle::getDeviceProp() const {
```

Comment:
Nitpick... maybe getDeviceProperties since then it nicely mirrors the real cuda call name?

Reply:
Makes sense. Renamed it.